libraries/SITL/SIM_JSON.cpp

```diff
@@ -318,6 +345,11 @@ void JSON::recv_fdm(const struct sitl_input &input)
         airspeed_pitot = constrain_float(velocity_air_bf * Vector3f(1.0f, 0.0f, 0.0f), 0.0f, 120.0f);
     }
 
+    num_motors = state.motor.rpm.length;
+    for (uint8_t i = 0; i < state.motor.rpm.length; i++) {
+        rpm[i] = state.motor.rpm.data[i];
```

Comment:
This looks like a good place for a memcpy.

Reply:
fixed.